assets/js/index.js

```diff
@@ -0,0 +1,157 @@
+// Assignment Code
+const generateBtn = document.querySelector("#generate");
+
+//function - get the password length from the user
+const getPasswordLength = () => {
+  //prompt user to give a userInput
+  //declare variable to receive user input - const -> not modified further down
+  const userInput = prompt(
+    "How long would you like your password to be? \n\nPlease enter a number between 8 and 128.\n",
+    ""
+  );
+  // check if userInput is a number
+  const passwordLength = parseInt(userInput);
```

Comment:
Always good to pass the second radix argument when using the `parseInt` function which is `10`. See documentation for more information on that.

```suggestion
  const passwordLength = parseInt(userInput, 10);
```